CityAssistant/CityAssistant.xcodeproj/project.pbxproj

```diff
@@ -12,7 +12,6 @@
 		BD163B852A47A27A0048AE2B /* POIMapView.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD163B842A47A27A0048AE2B /* POIMapView.swift */; };
 		BD163B872A47AE070048AE2B /* POICard.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD163B862A47AE070048AE2B /* POICard.swift */; };
 		BD163B892A47C08F0048AE2B /* LocationManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD163B882A47C08F0048AE2B /* LocationManager.swift */; };
-		BD163B8E2A49636D0048AE2B /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = BD163B8C2A49636D0048AE2B /* LaunchScreen.storyboard */; };
```

Comment:
Did you mean to remove this?

Reply:
I removed the Storyboards that were in there because we're using SwiftUI.